### DISCUSSION TITLE
Setting up my env to contribute to the Netty project

### DISCUSSION THREAD

**Discussion by fmisiak** [netty/netty#12160] [Q&A]
I'm novice to the Netty project (from a dev/contributor point of view).
I followed https://netty.io/wiki/setting-up-development-environment.html (for Linux/Ubuntu).
I checkout "4.1.74.Final" from the git repo.
I was able to successfully run:
`maven clean install -Dmaven.test.skip=true`

But when I include the unit tests in this build, I'm observing failures on "Netty/Handler" such as:
<block>
[ERROR] Failures: 
[ERROR]   AmazonCorrettoSslEngineTest>SSLEngineTest.testRSASSAPSS:4210->SSLEngineTest.rethrowIfNotNull:914 Expected no error
[ERROR]   AmazonCorrettoSslEngineTest>SSLEngineTest.testRSASSAPSS:4210->SSLEngineTest.rethrowIfNotNull:914 Expected no error
[ERROR]   AmazonCorrettoSslEngineTest>SSLEngineTest.testRSASSAPSS:4210->SSLEngineTest.rethrowIfNotNull:914 Expected no error
[ERROR]   AmazonCorrettoSslEngineTest>SSLEngineTest.testRSASSAPSS:4210->SSLEngineTest.rethrowIfNotNull:914 Expected no error
[ERROR]   AmazonCorrettoSslEngineTest>SSLEngineTest.testRSASSAPSS:4210->SSLEngineTest.rethrowIfNotNull:914 Expected no error
[ERROR]   AmazonCorrettoSslEngineTest>SSLEngineTest.testRSASSAPSS:4210->SSLEngineTest.rethrowIfNotNull:914 Expected no error
[ERROR]   ConscryptJdkSslEngineInteropTest>SSLEngineTest.testRSASSAPSS:4210->SSLEngineTest.rethrowIfNotNull:914 Expected no error
[ERROR]   ConscryptJdkSslEngineInteropTest>SSLEngineTest.testRSASSAPSS:4210->SSLEngineTest.rethrowIfNotNull:914 Expected no error
[ERROR]   ConscryptJdkSslEngineInteropTest>SSLEngineTest.testRSASSAPSS:4210->SSLEngineTest.rethrowIfNotNull:914 Expected no error
.....
(abridged)
</block>
Is there something obvious I am missing in my dev env?
Are these tests supposed to be disabled somehow?
Thank you for any feedback!

**Comment by normanmaurer** [comment]
Hmm this works for me here... Can you share the full log ?

**Comment by fmisiak** [comment]
Norman,
See attached maven log.
[mvn-external-maven-repo.log.gz](https://github.com/netty/netty/files/8226297/mvn-external-maven-repo.log.gz)
I really appreciate you taking the time to look into this.
Thank you.
